### ISSUE TITLE
Projections API

### ISSUE THREAD

**Issue by michael-schnell** [jen20/EventStore.Java#12]
Having a way to create, read, update and delete projections would be great.

**Comment by jen20** [comment]
Agreed, this would be a good thing to add. It can make use of the REST API for projections (there is no TCP management interface for projections). This is likely to be a V2 feature though - I think it's currently more important to get the core operations in.

**Comment by michael-schnell** [comment]
:+1:
